iroh-ffi/.gitignore

```diff
@@ -0,0 +1,4 @@
+/target
+/Cargo.lock
```

Comment:
can we make this part of the workspace please, then you shouldn't need these two lines